src/Clock.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Clock.scss';
+
+export class Clock extends React.Component {
+state = {
+  time: new Date(),
+}
+
+componentDidMount() {
+  setInterval(() => {
+    const update = new Date();
+
+    this.setState({ time: update });
```

Comment:
```suggestion
    this.setState({ time: new Date() });
```
do you need variable for that?